src/Assets/Animations/AnimationCustom.cpp

```diff
@@ -8,16 +8,18 @@
 #include <iostream>
 #include "AnimationNode.h"
 
-Transformation AnimationCustom::calculateTransform(const std::string& nodeName __attribute((unused)), float time, bool &isFound) const {
-    Transformation resultTransformation;
-
-    resultTransformation.setScale( animationNode->getScalingVector(time));
-    resultTransformation.setTranslate( animationNode->getPositionVector(time));
-    resultTransformation.setOrientation(animationNode->getRotationQuat(time));
+bool AnimationCustom::calculateTransform(const std::string& nodeName __attribute((unused)), float time, Transformation& transformation) const {
+    bool status = false;
+    if (nodes.find(nodeName) == nodes.end()) {
```

Comment:
There is no nodes map, and it is not used in custom animations. that part can be removed completely.